docs/getting_started.rst

```diff
@@ -71,15 +71,27 @@ Running Open3D tutorials
 
 Importing py3d module
 ----------------------
-By default, compiling Open3D will create a Python library under directory ``build/lib/`` or ``build/lib/Release/``.
-For quick sanity check, try importing the library using following python script:
 
-.. code-block:: python
+Compiling Open3D will create a Python library under directory ``build/lib/`` or ``build/lib/Release/``.
+Install/uninstall this module on Linux/Mac by running the utilities in ``util/scripts``.
 
-    # type "python" and use following script
-    import sys
-    sys.path.append("lib") # (or "lib/Release")
-    import py3d
+.. code-block:: sh
+
+    cd util/scripts
+
+    # install the Python module
+    ./install.sh
+
+    # uninstall the Python module
+    ./uninstall.sh
+
+For quick sanity check, try importing the library from the python interactive shell:
+
+.. code-block:: sh
+
+    python
+
+    >>> import py3d
```

Comment:
Please add more description for windows. Please check my comments in this PR.

Reply:
OK, I see, it works by default on Windows too. Awesome :).

Reply:
DONE